### PR TITLE
feat: command for db schema migrations

### LINKED ISSUE #25: Support DB schema migrations
We should support controlled schema migrations. We can use tools like https://github.com/golang-migrate/migrate or https://github.com/rubenv/sql-migrate

### PR DESCRIPTION
**Pull request by llorllale**
closes #25 

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Comment by rolsonquadras** [comment]
@llorllale This PR has conflicts after #31 - need to rebase.

**Comment by llorllale** [comment]
@rolsonquadras done

File: cmd/adapter-rest/migratecmd/migrate.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	dsn *dburl.URL
+	dir int
+}
+
+// GetMigrateCmd returns the cobra migrate command.
+func GetMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "DB migration.",
+		Long:  "Upgrades and downgrades SQL database schemas.",
```
Comment (rolsonquadras) [inline cmd/adapter-rest/migratecmd/migrate.go:42]: Just curious about the migration command - assume we have a DB running and multiple instances of adapter (for load balancing), from which instance are we gonna run the migration command? Are we gonna have one instance purely for sql db migration ?

@llorllale @troyronda @sudeshrshetty @fqutishat

Comment (llorllale) [reply cmd/adapter-rest/migratecmd/migrate.go:42]: This PR allows the edge-adapter to migrate DB schemas via the CLI only.

**Pull request merged**